.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
+#      - rpm-software-management/ci-dnf-stack
+    branches:
+      - master
+    vars:
+      dnf_master_copr: fbo/dnf-master
+    run: zuul-ci/run-tests.yaml
+    nodeset: fedora-31-vm
```

Comment:
Finally the nodeset expose to the job. This nodeset is a single node nodeset. The node is Fedora 31 Virtual Machine. https://pagure.io/fedora-zuul-jobs/blob/master/f/zuul.d/nodesets.yaml